src/containers/explore/EntityDataContainer.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  AppContentWrapper,
+  Spinner,
+  Typography
+} from 'lattice-ui-kit';
+import {
+  DataUtils,
+  ReduxUtils,
+  ValidationUtils,
+  useRequestState
+} from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import type { FQN, Organization, UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import EntityNeighborsContainer from './EntityNeighborsContainer';
+import {
+  EXPLORE_ENTITY_DATA,
+  exploreEntityData,
+  exploreEntityNeighbors
+} from './actions';
+
+import {
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  LinkButton
+} from '../../components';
+
+import { FQNS } from '../../core/edm/constants';
+import {
+  ENTITY_NEIGHBORS_MAP,
+  EXPLORE,
+  SELECTED_ENTITY_DATA
+} from '../../core/redux/constants';
+import { selectOrgDataSet, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { clipboardWriteText } from '../../utils';
+import { EntityDataGrid } from './components';
+
+const { getPropertyValue } = DataUtils;
+const { isPending } = ReduxUtils;
+const { isValidUUID } = ValidationUtils;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+`;
+
+type Props = {
+  dataSetDataRoute ?:string;
+  dataSetId :UUID;
+  entityKeyId :UUID;
+  isModal ?:boolean;
+  organizationId :UUID;
+  organizationRoute ?:string;
+};
```

Comment:
No change required. I prefer to inline these because the tooltip actually shows the props. For example ...

Not inline:
<img width="656" alt="Screen Shot 2021-06-07 at 7 30 20 PM" src="https://user-images.githubusercontent.com/440299/121113386-d5d6ac80-c7c6-11eb-8649-01744f07a7ba.png">

Inline:
<img width="554" alt="Screen Shot 2021-06-07 at 7 30 10 PM" src="https://user-images.githubusercontent.com/440299/121113383-d4a57f80-c7c6-11eb-8ca3-82313d167230.png">